02/app.js

```diff
@@ -0,0 +1,81 @@
+const form = document.querySelector('form');
+const login = document.getElementById('formLogin');
+const pass1 = document.getElementById('formPass1');
+const pass2 = document.getElementById('formPass2');
+const accept = document.getElementById('formAccept');
+const formInputs = [...form.elements];
+console.log(formInputs);
+
+form.addEventListener('submit', sendForm);
+
+let errors = [];
+
+
+function sendForm(e) {
+    errors = [];
+    e.preventDefault();
+    formInputs.forEach(Validation);
+
+    if (errors.length === 0) {
+        console.log('done');
+        window.confirm('Your form has been sent');
```

Comment:
Lepiej wykorzystać alert, bo nie potrzebujemy żadnego dodatkowego potwierdzenia od strony użytkownika - to tylko informacja dla niego.